robottelo/ui/base.py

```diff
@@ -186,7 +186,7 @@ def delete_entity(self, name, really, name_locator, del_locator,
         else:
             raise Exception("Could not search the entity '%s'" % name)
 
-    def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
+    def wait_until_element(self, locator, timeout=20, poll_frequency=0.5):
```

Comment:
Ok .. thats correct ! It will affect other tests! 
The issue was specific to search_entity and so increased the time timeout from search_entity method.